src/ray/object_manager/plasma/store.cc

```diff
@@ -544,15 +548,24 @@ bool PlasmaStore::IsObjectSpillable(const ObjectID &object_id) {
 
 void PlasmaStore::PrintAndRecordDebugDump() const {
   absl::MutexLock lock(&mutex_);
-  RecordMetrics();
   RAY_LOG(INFO) << GetDebugDump();
   stats_timer_ = execute_after(
       io_context_,
       [this]() { PrintAndRecordDebugDump(); },
       RayConfig::instance().event_stats_print_interval_ms());
 }
 
-void PlasmaStore::RecordMetrics() const { object_lifecycle_mgr_.RecordMetrics(); }
+void PlasmaStore::RecordMetrics() const {
```

Comment:
Can you rename this to `ScheduleRecordMetrics` or similar since it's no longer a one-shot thing?

Reply:
Ah good point.